app/components/team/Index.js

```diff
@@ -93,16 +93,16 @@ class TeamIndex extends React.PureComponent {
         <div>
           <Panel>
             <Panel.Section className="max-width-3">
-              <p>Teams allows you to group people, pipelines and permissions together.</p>
+              <p>Teams allows you to assign permissions to different groups of users and pipelines. You can create secret teams which are only visible to account admins and team members, or public teams that are visible to the your entire Buildkite organization.</p>
               <p>
                 <img
-                  src="//placekitten.com/800/480"
-                  width="400"
-                  height="240"
-                  alt=""
-                  title="This tiny kitten uses Teams - shouldn't you?"
+                  src={require('./preview.png')}
+                  alt="A screenshot of the teams functionality"
+                  title="Screenshot showing a list of 4 different teams, with different users and pipelines associated to each team."
+                  className="fit"
                 />
               </p>
+              <p>Once you’ve enabled teams, all your users and pipelines will be added to a new team called “Everyone” to ensure continued access to pipelines. You can customize, rename or delete the default “Everyone” team.</p>
```

Comment:
How about instead of 'ensure continued access' we have 'to maintain existing pipeline access'? I'm like a 4/10 on this though, so I really don't mind.